### DISCUSSION TITLE
Update Heroes

### DISCUSSION THREAD

**Discussion by Backxtar** [TeKrop/overfast-api#33] [Ideas]
Maybe you can add some parameters to the hero pages? Like HP, SHIELD, ARMOR, synergy with: `Kiriko, bla, bla`?
Also can you create an endpoint, where the information from /heroes/key is show for all heroes? I have to loop threw all heroes to get a list of all hero specs :)

Would be awesome :)! Im using your api to create a flutter app (ios + android)

Overall an awesome api!

**Comment by TeKrop** [comment]
Good suggestion, I'll do this in the context of the issue you created 😄 #34
Regarding your Flutter app, don't hesitate to implement some kind of local caching while retrieving hero data, as it will not change very often 😉
